enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -0,0 +1,219 @@
+---
+title: hypershift-audit-logs
+authors:
+  - aconway@redhat.com
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2023-03-31
+tracking-link:
+  - TBD
+see-also:
+  - /enhancements/cluster-logging/api-audit-log-policy.md
+---
+
+# Hypershift Audit Logs
+
+## Summary
+
+In a self contained cluster, users can forward application, infrastructure and audit logs.
+In a hypershift cluster, API server audit logs do not exist on the guest cluster,
+they are generated in Hypershift Control Plane (HCP) pods on the management cluster.
+
+This proposal deploys the existing cluster logging operator (CLO) on the management cluster.
+It will forward HCP audit logs according to a policy and destination configured on the guest cluster,
+and independently forward audit logs with the policy/ destination required by hypershift SD.
+
+**NOTE**: This proposal depends on [api-audit-log-policy](/api-audit-log-policy.md)
+
+## Motivation
+
+- Allow users and SREs with conflicting requirements to get the logs they need.
+- Forward API server logs from the management cluster, as they are not available on the guest cluster.
+- Replace the SD Splunk forwarder with the CLO which is more flexible and feature-rich.
+- Centralize log forwarding in the logging project, give hypershift team control of their logging needs.
+
+### User Stories
+
+#### As a cluster owner, I want to store and/or forward selected API audit logs to my chosen destinations.
+
+- configure API audit policy and forwarding using guest cluster resource.
+- have the desired audit logs forwarded from the management cluster to my destination.
+
+#### As a Hypershift SRE, I want to control plane audit logs to go to my chosen destination with my policy.
+
+- configure the SD internal audit policy
```

Comment:
```suggestion
- configure the SD internal audit policy
- configure API audit policy and forwarding using namespaced logging customer resources in the management cluster.
```

Calling out namespaced resources here as we must keep in mind that management clusters _are_ OSD clusters right now.  The audit forwarding of the management cluster should leverage the non-hypreshift solution documented in this PR.  The MC is hosting many HCP.  These must be forwarded as well and the policy and configuration of outputs must be independent of the MC configuration.  For example, we have different types of events possibly and we ship to a different index in splunk.